pkg/iptables/iptables.go

```diff
@@ -50,7 +50,7 @@ func renderRule(rule *Rules) []string {
 func AddRule(name string, source string, ignore []string, egressnetwork string) error {
 
 	var (
-		egressIntreface string
+		egressInterface string
 		//		egressIpAddress string
```

Comment:
I would remove it instead of commenting unused variable :)

Reply:
I agree, removed:)